src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -228,8 +228,7 @@
 .AddonReviewCard:not(.AddonReviewCard-slim)
   .AddonReviewCard-confirmDeleteDialog {
   /* stylelint-enable */
```

Comment:
do we still need this (and `/* stylelint-disable */`) ?

Reply:
We still need to disable it for line 211 (which is above this), but I changed it from a general disable to a single-line, specific rule disable.

I was also able to remove the requirement to disable it for this set of lines by combining them onto a single line. Thanks for the heads-up.